docs/docs/installation/alerts-reports.mdx

```diff
@@ -203,121 +216,11 @@ USER superset
 
 > Don't forget to set `WEBDRIVER_TYPE` and `WEBDRIVER_OPTION_ARGS` in your config if you use Chrome.
 
-### Summary of steps to turn on alerts and reporting:
```

Comment:
these steps are still valid from my understanding why are we removing them?

Reply:
These steps aren't specific to Alerts & Reports, it's mostly a stale guide to "how to deploy with docker-compose" which is now redundant with https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/ and not universally relevant here.  And it has hard-coded old content to boot, like referring to v1.0.1 and a whole docker-compose.yaml file that is out of date from https://github.com/apache/superset/blob/master/docker-compose.yml.

For instance, it starts with creating a Dockerfile and building an image - that's not necessary for Alerts & Reports.

I took the few pieces specific to Alerts & Reports or not mentioned on other docs pages and made sure they were covered above, e.g., explaining Celery worker & beat and noting that the process works in a Docker swarm environment .